ansible/roles/prosody/defaults/main.yml

```diff
@@ -101,7 +101,9 @@ prosody_enable_wait_for_host: false
 # prosody_log_level: change 'info' to 'debug' for debug logs
 prosody_focus_admins:
   - "focus@{{ prosody_auth_domain_name }}"
-prosody_focus_user_secret: replaceme
+prosody_focus_secret: "{{ secrets_jicofo_focus }}"
```

Comment:
Similar here, we should ensure the role can be run without the secrets file included, by defining `replaceme` defaults.